composer.json

```diff
@@ -21,10 +21,10 @@
         {"name": "Steve Müller", "email": "st.mueller@dzh-online.de"}
     ],
     "require": {
-        "php": "^7.1 || ^8.0",
+        "php": "^7.2 || ^8.0",
         "dealerdirect/phpcodesniffer-composer-installer": "^0.6.2 || ^0.7",
-        "slevomat/coding-standard": "^7.0.0",
-        "squizlabs/php_codesniffer": "^3.6.0"
+        "slevomat/coding-standard": "^8",
```

Comment:
This is not a proper version constraint, it should be `^8.0`.

Reply:
No, it could not be interpreted as >=.

It's 8<=X<9

![Screenshot_20220622-004640_Chrome](https://user-images.githubusercontent.com/327717/174902716-6bb18921-9ae1-45f9-9d71-8eb53c51631a.jpg)

Reply:
I agree, the constraint is perfectly fine.

Reply:
Composer respects semver. `^8` means `8<=X<9`. 

If you think otherwise, can you create an example?